Resources/Prototypes/Recipes/Construction/Graphs/structures/barricades.yml

```diff
@@ -53,3 +72,30 @@
           steps:
             - tool: Prying
               doAfter: 5
+
+- type: constructionGraph
+  id: BarricadeCovering
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: barricadecover
+          steps:
+            - material: WoodPlank
+              amount: 2
+              doAfter: 3
+    - node: barricadecover
+      entity: BarricadeBlock
+      edges:
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: MaterialWoodPlank1
+              amount: 1 #returns 1 less as one breaks
```

Comment:
Change this to 2; deconstructing should be the intended way to get materials back since it requires tools. See the previous comment.